src/main/java/io/divolte/server/BrowserEventData.java

```diff
@@ -41,6 +44,7 @@
     public final Optional<Integer> screenPixelHeight;
     public final Optional<Integer> devicePixelRatio;
     public final Function<String, Optional<String>> eventParameterProducer;
+    public final Supplier<Map<String,String>> eventParametersProducer;
 
```

Comment:
Do we still need to keep the eventParameterProducer field? It's kind of redundant now that we have a producer for all of the parameters.